crates/services/txpool_v2/src/service.rs

```diff
@@ -417,13 +464,11 @@ where
                 .inc();
         }
 
-        let verification = self.verification.clone();
-        let pool = self.pool.clone();
         let p2p = self.p2p.clone();
+        let verification = self.verification.clone();
```

Comment:
`Verification` could potentially be expensive to clone. I see that most of its fields are actually `Arc`s, expect the newly added blacklist, which is a collection of four hashsets.

Would it be good to try to keep the blacklist in `Arc` as well? Or maybe the entire "verification" should be behind `Arc`?